skt/executable.py

```diff
@@ -775,15 +790,17 @@ def load_config(args):
                             config.get('publisher', 'destination'),
                             config.get('publisher', 'baseurl')]
 
-    if config.has_section('runner') and not cfg.get('runner'):
-        rcfg = {}
-        for (key, val) in config.items('runner'):
-            if key != 'type':
-                rcfg[key] = val
-        cfg['runner'] = [config.get('runner', 'type'), rcfg]
-    elif cfg.get('runner'):
-        cfg['runner'] = [cfg.get('runner')[0],
-                         ast.literal_eval(cfg.get('runner')[1])]
+    # Check if the runner type is set on the command line
+    if cfg.get('_name') == 'run' and cfg.get('type'):
+        # Use the runner configuration from the command line
+        cfg['runner'] = {
+            'type': cfg.get('type'),
+            'jobtemplate': cfg.get('jobtemplate'),
+            'jobowner': cfg.get('jobowner')
+        }
+    elif config.has_section('runner'):
+        # Use the runner configuration from the configuration file
+        cfg['runner'] = config.items('runner')
```

Comment:
Could you make the command-line options overwrite specific settings from the config file, instead of all of them? E.g. allow setting just the jobtemplate, but not jobowner?

Reply:
Why would we want to ignore any settings coming from the config file?

Reply:
Ideally, I would like to be able to override any config file setting with a command-line option. E.g. if the config file says "jobtemplate=A", I should be able to override it with the "--beaker-job-template" option. At the same time I would like e.g. the "jobowner" specified in the config file to stay unchanged.

How I read the code above is that we either take all the parameters from the command line, or from the config file.

Reply:
I think I see what you're saying now. Let me see if I can rework some of the code there.